app/uk/gov/hmrc/nisp/services/NationalInsuranceService.scala

```diff
@@ -21,32 +21,32 @@ import uk.gov.hmrc.domain.Nino
 import uk.gov.hmrc.http.{HeaderCarrier, UpstreamErrorResponse}
 import uk.gov.hmrc.nisp.connectors.NationalInsuranceConnectorImpl
 import uk.gov.hmrc.nisp.models.StatePensionExclusion.{CopeStatePensionExclusion, ForbiddenStatePensionExclusion, OkStatePensionExclusion}
-import uk.gov.hmrc.nisp.models.{Exclusion, NationalInsuranceRecord}
+import uk.gov.hmrc.nisp.models.{Exclusion, NationalInsuranceRecord, StatePensionExclusion, StatePensionExclusionFilter, StatePensionExclusionFiltered, StatePensionExclusionFilteredWithCopeDate}
```

Comment:
I'm thinking we don't use all these imports. SO could tidy this up by using ._ or removing the ones we don't use